ksql-serde/src/test/java/io/confluent/ksql/serde/delimited/KsqlDelimitedDeserializerTest.java

```diff
@@ -71,4 +72,33 @@ public void shouldDeserializeJsonCorrectlyWithRedundantFields() throws JsonProce
     Assert.assertNull(genericRow.getColumns().get(3));
   }
 
+  @Test
+  public void shouldDeserializeDelimitedCorrectlyWithOtherDelimiters() {
+    final String rowString = "1511897796092\t1\titem_1\t10.0\r\n";
+
+    final KsqlDelimitedDeserializer ksqlJsonDeserializer = new KsqlDelimitedDeserializer(orderSchema, CSVFormat.TDF);
+
+    final GenericRow genericRow = ksqlJsonDeserializer.deserialize("", rowString.getBytes());
+    assertThat(genericRow.getColumns().size(), equalTo(4));
+    assertThat((Long) genericRow.getColumns().get(0), equalTo(1511897796092L));
+    assertThat((Long) genericRow.getColumns().get(1), equalTo(1L));
+    assertThat((String) genericRow.getColumns().get(2), equalTo("item_1"));
+    assertThat((Double) genericRow.getColumns().get(3), equalTo(10.0));
+  }
+
+  @Test
+  public void shouldDeserializeJsonCorrectlyWithRedundantFieldsWithOtherDelimiters() throws JsonProcessingException {
```

Comment:
Looks like some of this has been copied from a Json serde test. Can you clean that up?

Reply:
this was actually copied from `shouldDeserializeJsonCorrectlyWithRedundantFields ` function above.. I'll remove it and add another with a different type

Reply:
fixed in d5ebf50e...ef32d83e